### PR TITLE
Research facility upgrade

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary

```SUMMARY: Balance "Added copious amounts of chaos to research facilities"```

#### Purpose of change
The research center is currently a source of some pretty good loot, until the rest of the world catches up with realistic spawn rates and some of the items I've put there. However, it's also currently underspawning monsters and hazards, making it too easy to crack that sweet place open.

#### Describe the solution
Added map extras, allowing portals and craters to spawn there. This adds to the chaos.
Greatly increased monster spawn rates and adjusted spawn lists to have more threatening zombies (and also to reduce the number of 'boring' zombies spawning). This place would have been one of the last bastions, where people were frenziedly trying to figure out how to stop the apocalypse... it makes sense for it to be loaded with zombies.

#### Describe alternatives you've considered
I would like to add many unique map extras to this location, but that takes c++ so will have to wait.

#### Additional context
aw crap
image

uh oh
image

This is what happens when you don't balance the ultracentrifuge, Josh.
image

Yep, I am pretty sure this is where the apocalypse started.
image

#### Additional additional context

This is what happens when you call things too easy.

### CODE REVIEW

**Comment by I-am-Erk** [comment]
Jenkins Rebuild

File: data/json/regional_map_settings.json [hunk 1]
```diff
@@ -544,6 +544,31 @@
...
+      },
+      "research_facility_lot": {
+        "chance": 3,
+        "extras": {
+          "mx_helicopter": 40,
+          "mx_military": 65,
+          "mx_collegekids": 1,
+          "mx_portal": 160,
+          "mx_crater": 180,
+          "mx_fumarole": 10,
```
Comment (Inglonias) [inline data/json/regional_map_settings.json]: Is it possible for fumaroles to burn down the whole building, and if so, is that ok?

Comment (I-am-Erk) [reply data/json/regional_map_settings.json]: I don't see why not, to both

Comment (Inglonias) [reply data/json/regional_map_settings.json]: That's fine then.

**Comment by I-am-Erk** [comment]
Jenkins Rebuild

**Comment by I-am-Erk** [comment]
jenkins rebuild

**Comment by Inglonias** [comment]
Jenkins is pretty backed up right now on account of we had two primary builds today alone, and another is already in the queue. Asking again won't help, I'm afraid.

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/research-facility-is-the-new-necropolis/20445/13

**Pull request merged**